mainapp/proj.android-studio/app/src/main/res/layout/activity_splash_screen.xml

```diff
@@ -25,8 +25,8 @@
 
     <TextView
         android:id="@+id/oneTimeExtractionMessage"
-        android:layout_width="599dp"
-        android:layout_height="40dp"
+        android:layout_width="653dp"
+        android:layout_height="55dp"
```

Comment:
Are these values working for different screens?

Reply:
Updated the styles and checked on different resolutions and everything is working fine